cmd/kubeadm/app/cmd/phases/reset/cleanupnode.go

```diff
@@ -91,12 +103,16 @@ func runCleanupNode(c workflow.RunData) error {
 	if certsDir != kubeadmapiv1.DefaultCertificatesDir {
 		klog.Warningf("[reset] WARNING: Cleaning a non-default certificates directory: %q\n", certsDir)
 	}
-	resetConfigDir(kubeadmconstants.KubernetesDir, certsDir)
+	resetConfigDir(kubeadmconstants.KubernetesDir, certsDir, r.DryRun())
```

Comment:
i think we can just remove the Infoln on line 90:
`klog.V(1).Infoln("[reset] Removing contents from the config and pki directories")`

i don't think it contributes anything because later the directories are being printed.